src/components/Navbar/Navbar.scss

```diff
@@ -1,3 +1,42 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
 
-/* Delete these lines when you begin development */
+.Navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: var(--extra-small-spacer) var(--extra-large-spacer);
+  margin-bottom: -58px;
+
+  background: var(--white);
+  filter: drop-shadow(0 10px 15px rgba(0, 0, 0, 0.3));
+  border-radius: 0  0 25px 25px;
+
+  @include responsiveness.respond(phone) {
+    flex-direction: column;
+  }
+
+  &__title {
+    @include typography.title-text;
+    color: var(--wedgewood);
+
+    @include responsiveness.font($responsive: 3.8vw, $min: 28px, $max: 35px);
+  }
+
+  &__buttonContainer {
+    display: flex;
```

Comment:
Perhaps we can stack the buttons vertically for small screens? Or, we should ask design for guidance. 

<img src="https://user-images.githubusercontent.com/20177171/99858993-a4a4d080-2b43-11eb-9609-72c33f3d4623.png" width="250" />

Reply:
We shouldn't need to repeat `CruzHacks 2021` in the navbar, I second consulting design on this

Reply:
@w-gao just for `410px` or overall?

I kind of like this one: 
![image](https://user-images.githubusercontent.com/33771623/99866628-80f38180-2b67-11eb-88f1-eaf10316eb99.png)

Reply:
Oops, did not see this mention. This looks great, the updated version LGTM as well.